project_setup/README.md

```diff
@@ -4,6 +4,32 @@ While the specific setup for each 18F project varies widely, there are certain
 elements that should be present on every project. This document aims to detail
 those elements and provide suggested tools and resources.
 
+### Initialization Checklist
+
+This list is aspirational, but a good place to start. We will try to include
+all in upcoming project templates; until then, do your best!
+
+1. Common project files (consider using [`18f init`](https://github.com/18F/18f-cli#18f-init)):
+    1. LICENSE.md
+    1. CONTRIBUTING.md
+    1. README.md
+    1. .about.yml
+1. Project files (e.g. package.json, setup.py) if relevant
+1. Build scripts (e.g. grunt, rake, manage.py) if relevant
+1. Dependency descriptions (e.g. Gemfiles, requirements.txt) if relevant.
+Don't forget to [pin](https://pages.18f.gov/before-you-ship/infrastructure/pinning-dependencies/)
+them
+1. Cloud.gov manifests (one per environment) if applicable. See [fec-cms](https://github.com/18F/fec-cms) for a great example
+1. Linter setup (straight [flake8](http://flake8.pycqa.org/en/latest/), [rubocop](https://github.com/18F/development-guide/blob/master/ruby/.rubocop.yml), [eslint](https://github.com/airbnb/javascript/blob/master/linters/.eslintrc))
```

Comment:
May make sense to eventually have this point to language-specific dev guides rather than specific linters (assuming our lang-specific linter prefs will change over time)